scene/scene-properties-expressions/README.md

```diff
@@ -0,0 +1,27 @@
+#Scene Properties Expressions#
+Demonstrates how to update the orientation of a graphic using scene property rotation expressions.
+
+##How to use the sample##
+Move the heading and pitch sliders to change the cone's orientation.
+
+![](ScenePropertiesExpressions.png)
+
+##How it works##
+To update a graphic's orientation using expressions:
+
+1. Create a new `GraphicsOverlay`.
+2. Create a `SimpleRenderer` and set expressions on its scene properties: `renderer.getSceneProperties()
```

Comment:
kind of makes it look like a static method when you do that though. We should come up with a standard and update the creating a sample guide.